src/conda/env_precip_buoy_diag.yml

```diff
@@ -0,0 +1,19 @@
+name: _MDTF_precip_buoy_diag
```

Comment:
- Please double-check that you're only including the dependencies used by your POD (eg. I don't see where you use the NCO utilities. 
- For future compatibility, please specify at least the minor version number of all dependencies (eg. x.y, or x.y.z where it makes sense to do so) so that we have a known-good environment for running your POD.

Reply:
- Removed unused dependencies and included package version numbers in https://github.com/ahmedfiaz/MDTF-diagnostics/commit/975cbfe2729365c0f1fb683d2e6cd7b7678b5594
-While testing, conda failed to install _MDTF_base. Framework then exited without NCO utilities. So included NCO in precip_buoy_diag.yml. Removed this now https://github.com/ahmedfiaz/MDTF-diagnostics/commit/617a3dc212bbe9e041bf1eaf6791ac61751bf237